lib/rubocop/rails/schema_loader.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Rails
+    # It loads db/schema.rb and return Schema object.
+    # Cops refers database schema information with this module.
+    module SchemaLoader
```

Comment:
This is the schema loader.

I implemented it as a mixin at first, but it was a bad idea. So now it is a non-mixin module.

If it is a mixin, the `@schema` cache only works for a single cop and single file. So it needs to parse `db/schema.rb` many times.
But if the non-mixin module, just once parsing is enough.